README.md

```diff
@@ -2,7 +2,9 @@
 
 ## hadoop-azure-token-providers
 
-The [`hadoop-azure` library](https://hadoop.apache.org/docs/stable/hadoop-azure/abfs.html) supports supplying a [custom token provider](https://hadoop.apache.org/docs/stable/hadoop-azure/abfs.html#Custom_OAuth_2.0_Token_Provider) for getting accessing tokens for Azure Storage.
+The package was built in order to enable a simple way of accessing data in Azure Data Lake when developing a PySpark solution locally in a Dev Container.
```

Comment:
I take it that's Data Lake Gen 2?

Reply:
What about OneLake?

Reply:
Yes Gen2. No idea about OneLake - does that use the same protocol?

Reply:
That's what I'm asking you to find out! Maybe @edfreeman or @Smartitect knows?

Reply:
It looks like it does use the same protocol, but I would like to test it out before stating that it works

Reply:
Seems to work fine with OneLake too, have updated the text